kuma/attachments/models.py

```diff
@@ -8,10 +8,37 @@
 from django.utils.functional import cached_property
 from django.utils.translation import ugettext_lazy as _
 from django_mysql.models import Model as MySQLModel
+from storages.backends.s3boto3 import S3Boto3Storage
 
 from .utils import attachment_upload_to, full_attachment_url
 
 
+class AttachmentStorage(S3Boto3Storage):
+    def __init__(self, *args, **kwargs):
+        configuration = dict(
```

Comment:
Just for my understanding: Why are we subclassing S3Boto3Storage instead of passing the configuration in when instantiating?

Reply:
You can find the whole details in the comments marked "resolved" above, but TL;DR is that whatever you pass directly to the FileField gets included as part of the model migration, which seemed super awful. E.g., it ends up embedding the AWS credentials in the migration.

Subclassing avoids that, since those settings are hidden inside the constructor.

Reply:
Ah oh, I see. Thanks for the context, makes sense!